test/e2e/network/topology_hints.go

```diff
@@ -41,7 +40,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = common.SIGDescribe(feature.TopologyHints, func() {
```

Comment:
`Feature` in the e2e framework terminology indicates `this test needs to enable something to work, does not run in a normal cluster`

Reply:
this test requires a cluster to have zones so we need to add the Feature tag or it will be picked by any normal test